src/Feature/Catalog/tests/Controllers/SearchControllerTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Feature.Catalog.Tests.Controllers
+{
+    using System;
+
+    using Catalog.Models.Requests;
+
+    using Foundation.Commerce.Models;
+    using Foundation.Commerce.Models.Entities;
+
+    using ModelMappers;
+
+    using NSubstitute;
+
+    using Wooli.Feature.Catalog.Controllers;
+    using Wooli.Foundation.Commerce.Services.Catalog;
+
+    using Xunit;
+
+    public class SearchControllerTests
+    {
+        private readonly IProductSearchService productSearchService;
```

Comment:
Why there's new line between private fields?

Reply:
Because this fields are logically different, so we split them by empty line to show this difference in formatting.